README.md

```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
 * Add unit tests
 * Add `<ReferenceMany>` field
 * Add complex field & input types
+
+## Contributing
```

Comment:
How about putting that in a `CONTRIBUTING.md` file? Github / Gitlab have special integration features for this.